app/styles/onboarding-card.module.css

```diff
@@ -70,14 +72,18 @@
 
 @media (max-width: 625px) {
     .onboarding-card{
-        min-height: 30vh;
+        /* min-height: 30vh; */
```

Comment:
remove comments

Reply:
Done with changes can you please check my latest commit